frontend/src/components/job_form/job_form.jsx

```diff
@@ -0,0 +1,138 @@
+import { BasicDateTimePicker } from "./time_picker"
+import { Container, Form, Button } from 'react-bootstrap';
+import { useState } from 'react';
+import toast from 'react-hot-toast';
+
+export const JobForm = () => {
+  const [jobTitle, setJobTitle] = useState('');
```

Comment:
I think it could make sense to just have the title be "jobtype @ address".
Once a worker accepts a job then they will have access to the comment (it could contain private information about the job for example).